proto/table.proto

```diff
@@ -21,7 +21,16 @@ message CreateTableCommand {
     string type = 2;
     string length = 3;
     bool nullable = 4;
-    string default = 5;
+    // string cannot be null!
+    bool useDefault = 5;
+    string default = 6;
```

Comment:
pridal jsem `useDefault`, protoze `default` nemuze byt `null`

Reply:
tak string default je prázdný string, takže imho pokud ti dojde `'\'\''` tak chceš default empty string když chceš null tak ti dojde `'null'` jako string když nechceš nic tak ti dojde `''` takže bych udělal:
```
$options = [
                'length' => $column->getLength(),
                'nullable' => $column->getNullable(),
                'isLatin' => $columnMeta ? $columnMeta->getIsLatin() : false,
]
if($column->getDefault() === '')
               $options['default'] = $column->getDefault();
```

Reply:
no, prave ze muses klidne chtit i prazdny string `''`, i kdyz je to divny...

Reply:
Když chceš prázdný string tak by to connection měla poslat jako `'\'\''` jinak to nebude fungovat https://github.com/keboola/php-datatypes/blob/master/src/Definition/Teradata.php#L357

Reply:
aj pokud chceš default jako string tak to musí dojít jako `\'my-string\'`

Reply:
v tom TD datatypes na to chybý test,ale řeší to connection. Je to vidět v třeba v testech pro synapse tady je to vidět https://github.com/keboola/php-datatypes/blob/master/tests/SynapseDatatypeTest.php#L255

Reply:
super point, to by me nanapadlo, upravim

Reply:
upraveno